src/app/digital/views/components/NOTGateView.ts

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable unicorn/filename-case */
+import {DEFAULT_BORDER_COLOR, DEFAULT_BORDER_WIDTH, DEFAULT_CURVE_BORDER_WIDTH, SELECTED_BORDER_COLOR} from "core/utils/Constants";
+
+import {V} from "Vector";
+
+import {NOTGate} from "core/models/types/digital";
+
+import {ComponentView} from "core/views/ComponentView";
+
+import {DigitalViewInfo} from "../DigitalViewInfo";
+
+
+export class NOTGateView extends ComponentView<NOTGate, DigitalViewInfo> {
```

Comment:
Note that we don't really need this custom view since it just calls the `ComponentView` constructor only, so you can change it in `src/app/digital/views/index.ts` to do `new ComponentView(c, o, V(1, 1), "buf.svg")` and then delete this file